extensions/todolist@bsaleil.org/Makefile.am

```diff
@@ -1,7 +1,7 @@
 EXTENSION_ID = todolist@bsaleil.org
 SCHEMA_ID = todolist
 
-EXTRA_MODULES = utils.js prefs.js
+EXTRA_MODULES = utils.js prefs.js todolist.js
```

Comment:
@LiliC Did you update from upstream for this changeset? If not we'll want to make a more localized fix.